tests/test_tasks.py

```diff
@@ -383,3 +403,85 @@ def ignore_call_to_send_delete_notice(self):
         (delete.send_delete_notice(
                 mox.IsA(tasks.CleanUpInTestMode), mox.IsA(model.Person)).
             MultipleTimes())
+
+
+class NotifyBadReviewStatusTests(unittest.TestCase):
+    handler = None
+
+    def setUp(self):
+        self.testbed = testbed.Testbed()
+        self.testbed.activate()
+        self.testbed.init_user_stub()
+        self.testbed.init_datastore_v3_stub()
+        self.testbed.init_memcache_stub()
+        self.testbed.init_mail_stub()
+        # root_path must be set the the location of queue.yaml.
+        # Otherwise, only the 'default' queue will be available.
+        path_to_app = os.path.join(os.path.dirname(__file__), '../app')
+        self.testbed.init_taskqueue_stub(root_path=path_to_app)
+
+        model.Repo(key_name='haiti').put()
+
+    def tearDown(self):
+        self.testbed.deactivate()
+
+    def get_handler(self):
+        if not self.handler:
+            self.handler = test_handler.initialize_handler(
+                handler_class=tasks.NotifyBadReviewStatus,
+                action=tasks.NotifyBadReviewStatus.ACTION,
+                repo='haiti', environ=None, params=None)
+        return self.handler
+
+    def clear_db_config(self):
+        [db.delete(entry) for entry in db.GqlQuery('SELECT * FROM ConfigEntry')]
```

Comment:
Thanks. I am kinda new to GAE, so its very helpful for me to get such feedbacks;)